python/paddle/dataset/conll05.py

```diff
@@ -24,11 +24,12 @@
 import gzip
 import itertools
 import paddle.dataset.common
-from six.moves import zip
+import paddle.fluid.compat as cpt
```

Comment:
It is very bad to cross-reference `paddle.fluid` in `paddle.dataset`.

Reply:
Maybe `paddle.compat` or `paddle.common.compat` is better.

Reply:
Maybe paddle.compat is better,  I will change this in next commit